.github/workflows/test.yml

```diff
@@ -1,6 +1,9 @@
 name: Test
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
When you push a commit or create a pull request, this workflow always runs.
What kind of use-case do you expect for workflow_dispatch?

Reply:
It's when you want to trigger a CI run without pushing a commit or creating a PR.

It's not a common use case, but can sometimes be handy and it helps to have the button enabled for when it's needed.